lib/vdsm/storage/exception.py

```diff
@@ -1349,6 +1349,22 @@ class VolumeGroupDoesNotExist(StorageException):
     code = 506
```

Comment:
Note for the commit message - title should be:

    storage.exception: Add error to VolumeGroupDoesNotExist